setup.py

```diff
@@ -9,7 +9,7 @@
     python_requires=">=3.8",
     install_requires=[
         "watertap @ https://github.com/watertap-org/watertap/archive/refs/tags/pr967.zip",
-        "pyomo <= 6.5",
+        "pyomo == 6.6.1",
```

Comment:
```suggestion
        "pyomo <= 6.6.1",
```
Did this not work?

Reply:
It works but I was trying to be specific with the version.